tests_e2e/tests/scripts/agent-python

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
```

Comment:
My next PR will include code to deploy pypy to the test VMs, in order to use the same Python version in all of our tests. Please sync with me offline.